polyn_elixir_client/lib/mix/tasks/polyn.migrate.ex

```diff
@@ -11,8 +11,6 @@ defmodule Mix.Tasks.Polyn.Migrate do
   def run(args) do
     {:ok, _apps} = Application.ensure_all_started(:polyn)
 
```

Comment:
Out of curiosity, how did you manage to avoid waiting on the connection to come up?